LLama/runtimes/build/LLamaSharp.Backend.Cpu.nuspec

```diff
@@ -13,38 +13,42 @@
     <releaseNotes></releaseNotes>
     <copyright>Copyright 2023 The llama.cpp Authors. All rights reserved.</copyright>
     <tags>LLamaSharp LLama LLM GPT AI ChatBot SciSharp</tags>
+    <contentFiles>
+      <files include="any\any\llama\**\*.*" buildAction="None" copyToOutput="true" />
```

Comment:
May I ask why `any/any/llama` is used here, instead of using `runtime`?

Reply:
When publishing a "self contained" app The runtime folder hierarchy seems to be flattened. That means that all of the files are put into the same folder, which means there are duplicates and therefore the build fails. By putting the files in `content/any/any` there's no special handling and so the folder hierarchy is preserved.

At least... that's the theory!